package.json

```diff
@@ -24,6 +24,7 @@
   "dependencies": {
     "@babel/core": "^7.16.0",
     "@babel/preset-env": "^7.16.4",
-    "babel-jest": "^27.3.1"
+    "babel-jest": "^27.3.1",
+    "moment": "^2.29.1"
```

Comment:
What do you need moment.js for? This library is no longer used. Please remove it